tests/test_evaluate.py

```diff
@@ -9,7 +9,7 @@
 from fusillade.directory.authorization import get_resource_authz_parameters
 from fusillade.utils.authorize import evaluate_policy
 from fusillade.directory import cleanup_directory, cleanup_schema, User, clear_cd, Role
-from fusillade.errors import FusilladeForbiddenException
+from fusillade.errors import FusilladeForbiddenException, ResourceNotFound, AuthorizationException
```

Comment:
```suggestion
from fusillade.errors import ResourceNotFound, AuthorizationException
```

Reply:
Is the `FusilladeForbiddenException` still useful, or should it be replaced with other, more specific exceptions? (`FusilladeForbiddenException` appears in a few spots in `fusillade/utils/authorize.py`)

Reply:
using FusilladeForbiddenException for this section of code was a mistake. This error is for when the requester is not authorized. For the evaluation endpoint the requester is asking if another user/principal is authorized to do something. Therefore if the user is not authorized we return "200 and result false" to the requester not an http 404 error.